### DISCUSSION TITLE
full text search with serverpod

### DISCUSSION THREAD

**Discussion by abcnow** [serverpod/serverpod#1036] [Q&A] [answered]
I am trying to implement a full text search and I wonder what is the best way to do this with serverpod? Imagine that you have about 100k books and that you want users to search content and generated topics... thanks in advance!!

**Comment by andriyndev** [comment, marked as answer]
I'm currently taking the following course
https://www.coursera.org/learn/json-natural-language-processing-postgresql
which explains in detail how to perform full-text search in PostgreSQL. The answer is: use inverted indices. You can check the course (watching the videos is free) or google "inverted index in postgresql" (Serverpod uses PostgreSQL).
